static-analysis/assert-one-of.php.disabled

```diff
@@ -0,0 +1,25 @@
+<?php
+
+namespace Webmozart\Assert\StaticAnalysis\AssertIsInstanceOfAny;
+
+use Webmozart\Assert\Assert;
+
+class A
+{
+}
+
+class B
+{
+}
+
+/**
+ * @param mixed $value
+ *
+ * @return A|B
```

Comment:
Similar to the above: does `@psalm-assert` work with union types?

Very similar to https://github.com/webmozart/assert/pull/102/files#r291697765